src/canvas-extractor/README.md

```diff
@@ -31,7 +31,7 @@ the this tool and its output files, please see the main repository
 
       ```bash
       poetry run python edfi_canvas_extractor -b [canvas url] -a [api token]
-          -s [start date range] -e [end date range]
+          -s [start date range] -e [end date range] --feature [feature1 feature2 feature3]
```

Comment:
Eric had specifically said that he wanted the examples to show turning on assignments.
```suggestion
          -s [start date range] -e [end date range] -f assignments
```